data/dataDefaulter.js

```diff
@@ -0,0 +1,51 @@
+function addDefaultsToResults(
+    scrapedResults,
+    cachedResults,
+    secondsOfTolerance
+) {
+    if (!(cachedResults && cachedResults.length)) {
+        return scrapedResults;
+    } else {
+        const combinedResults = [];
+        const scrapeGrouping = {};
+        scrapedResults.forEach((result) => {
+            combinedResults.push(result);
+            scrapeGrouping[generateKey(result)] = 1;
```

Comment:
I don't know what a `fooGrouping` is. Perhaps `resultFound` or `scrapedResultFound`?